acceptance/openstack/compute/v2/servers_test.go

```diff
@@ -515,13 +515,7 @@ func TestServersTags(t *testing.T) {
 	// All the following calls should work with "2.26" microversion.
 	client.Microversion = "2.26"
 
-	// Check server tags in body.
-	type serverWithTagsExt struct {
-		servers.Server
-		tags.ServerTagsExt
-	}
-
-	serverWithTags := serverWithTagsExt{}
+	serverWithTags := servers.Server{}
```

Comment:
Per other comments, please revert this change and rebase. After the rebase, I believe this file can go unmodified. Let me know if you have any questions.